contribution/testing_guide.md

```diff
@@ -1,20 +1,17 @@
 # Testing Guide
 
 There are two ways to check the functionalities of KubeArmor: 1) testing KubeArmor manually and 2) using the testing framework.
+# 0. Make sure Kubernetes cluster is running 
 
-# 0. Make sure that the annotation controller is installed on the cluster (Applicable for Steps 1 and 2)
-
-- To install the controller from KubeArmor docker repository to your cluster run
-
+Although there are many ways to run a Kubernetes cluster (like minikube or kind) but it will not work with locally developed KubeArmor. KubeArmor needs to be on the same node as where the kubernetes nodes exist. If you will try to do this it will not identify your node since minikube and kind use virtualised node. You would either need to build your images and deploy them into these clusters or simply use `k3s` or `kubeadm`. if you are new to these terms then the most easiest way to do this is by following this guide: [K3s installation guide](k3s/README.md)
 ```text
-$ cd KubeArmor/pkg/KubeArmorAnnotation
-~/KubeArmor/pkg/KubeArmorAnnotation$ make deploy
-```
-- To install the controller (local version) to your cluster run
-
-```text
-$ cd KubeArmor/pkg/KubeArmorAnnotation
-~/KubeArmor/pkg/KubeArmorAnnotation$ make docker-build deploy
+$ cd KubeArmor/KubeArmor
+~/KubeArmor/KubeArmor$ make clean && make
+~/KubeArmor/KubeArmor$ sudo -E ./kubearmor -gRPC=[gRPC port number]
+                                           -logPath=[log file path]
+                                           -enableKubeArmorPolicy=[true|false]
+                                           -enableKubeArmorHostPolicy=[true|false]
+~/KubeArmor/tests$ make run
```

Comment:
```suggestion
~/KubeArmor/KubeArmor$ make run
```

make run automatically handles both the steps.

Noting we are talking about running KubeArmor here not tests. So please change this under Testing KubeArmor 1.2 and 1.3 not here

Reply:
Sorry for the confusing review.